app/models/user.rb

```diff
@@ -4,15 +4,16 @@ class User < ActiveRecord::Base
   # :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :timeoutable,
-         :token_authenticatable, :bearer_token_authenticatable
+         # :token_authenticatable,
```

Comment:
`token_authenticatable` is deprecated. I commented it out but didn't remove it totally because I'm not 100% sure devise is working without it. The plan is to remove this line eventually once that is confirmed.